test/errors/incompatibleRangeBounds-brief.good

```diff
@@ -0,0 +1,4 @@
+error in incompatibleRangeBounds.chpl:3: upper and lower bounds of range expression have incompatible types
```

Comment:
I think we can skip specifying upper and lower and safely shorten this to `bounds of range expression have incompatible types` without losing clarity.